test.sh

```diff
@@ -7,52 +7,131 @@ GREEN="\e[32m"
 MAGENTA="\e[95m"
 RESET="\e[0m"
 
+# This function will be called when running this script
+function main(){
+    # This function will be called when running this script
+    run_tests_in_development;
+    run_tests_in_production;
+}
 
-# development or production?
-if [[ $(ls -l docker-compose.yml) == *"production.yml"* ]]; then
```

Comment:
I think, you miss understood the function of this. This should run production tests if you are in production mode and development tests, if the application is in development mode. Maybe it's useful, to run all tests in development mode to test production configuration too. But development tests should not be run on productive systems.